src/main/java/net/masterthought/cucumber/json/Feature.java

```diff
@@ -104,9 +104,9 @@ public Status getStatus() {
     }
 
     public String getName() {
-        return StringUtils.isNotEmpty(name) ? getStatus().toHtmlClass()
-                + "<div class=\"feature-line\"><span class=\"feature-keyword\">" + keyword + ":</span> " + name
-                + "</div></div>" : "";
+        return StringUtils.isNotEmpty(name) ? getStatus().toHtmlClass("")
+                + "<div class=\"feature-line\"><span class=\"feature-keyword\">" + keyword + ":</span> " + name +
+        "<div style=\"float: right;\"><span class=\"expanded\" onclick=\"hideall();\">&nbsp;&nbsp;&nbsp;</span><span class=\"collapsed\" onclick=\"showall();\">&nbsp;&nbsp;&nbsp;</span></div></div>"+"</div>" : "";
```

Comment:
I agree with you. On the other hand I believe the most convenient would be to have all scenario closed as long as they pass - for failure scenario should be opened to see details and check which step failed

Can you do this?